config/standard/contact-summary.templated.js

```diff
@@ -1,16 +1,30 @@
+var now = extras.now;
+var MS_IN_DAY = extras.MS_IN_DAY;
+var DAYS_IN_PNC = extras.DAYS_IN_PNC;
+var IMMUNIZATION_LIST = extras.IMMUNIZATION_LIST;
+var isCoveredByUseCaseInLineage = extras.isCoveredByUseCaseInLineage;
+var getTreatmentProgram = extras.getTreatmentProgram;
```

Comment:
It's also interesting some items are explicitly called from the extras namespace while others are aliased (you probably found it as is)

Reply:
I aliased anything used more than twice, but explicitly used extras for anything used less than or exactly twice.